test/simplifyExpression/functionsSearch/absoluteValue.test.js

```diff
@@ -10,6 +10,9 @@ describe('abs', function () {
   const tests = [
     ['abs(4)', '4'],
     ['abs(-5)', '5'],
+    ['abs(-x)', 'x'],
+    ['abs(-xy)', 'xy'],
```

Comment:
Oh man, good point! I was basing this off of Cymath and didn't think it through. Ok, will close this.